setup.cfg

```diff
@@ -8,6 +8,7 @@ url = https://github.com/pytest-dev/pytest-xprocess/
 long_description = file: README.rst
 description = A pytest plugin for managing processes across test runs.
 classifiers=
+    Framework :: Pytest
```

Comment:
👀

Reply:
Well, I came across this during some [doc-fun](https://docs.pytest.org/en/latest/writing_plugins.html#making-your-plugin-installable-by-others) and since I was already here...